entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Compile the assets
+bundle exec rake assets:precompile
```

Comment:
This should be done once, when the container is built. Putting it here will mean that the assets are re-built every time the container restarts